dashboard/public/vigenere/vigenere.css

```diff
@@ -0,0 +1,96 @@
+textarea, input, .textarea {
+  font-family: monospace;
+  font-size: 14px;
+}
+
+.btn-group .btn {
+  margin-right: 0;
+}
+
+#left-column fieldset {
+  margin-bottom: 5px;
+  margin-left: 0px;
+}
+
+#left-column > div {
+  /*padding: 5px 10px;*/
+  /*border: solid 1px #5b6770;*/
+  /*box-shadow: 5px 5px 10px #5b6770;*/
+  /*margin-bottom: 10px;*/
```

Comment:
Feel free to toggle 'em on and off in developer tools; I'd love to hear your opinions